quantum/rgb_matrix/animations/pixel_fractal_anim.h

```diff
@@ -31,6 +31,10 @@ static bool PIXEL_FRACTAL(effect_params_t* params) {
 
     inline uint32_t interval(void) { return 3000 / scale16by8(qadd8(rgb_matrix_config.speed, 16), 16); }
 
+    if (params->init) {
+        rgb_matrix_set_color_all(0, 0, 0);
```

Comment:
Hmm, I wonder whether clearing all LEDs like that is the proper behavior here — what if some LEDs are excluded from effect usage by flags?

Although it seems that the `DIGITAL_RAIN` and `TYPING_HEATMAP` effects also do the same thing, so maybe that's OK, and flag users just need to be aware that changing the effect may result in clearing the excluded LEDs.